source/core/text-search-operators.txt

```diff
@@ -39,6 +39,51 @@ operators, including restrictions and behavior, see:
 
 - :expression:`$meta` projection operator
 
+Exact Phrase
```

Comment:
I'm a little concerned about the discoverability of these examples. It seems slightly out of place to have them tucked under the "Query Framework" section. These seem like critical use cases we should be highlighting to users.

What if we:

* Moved these examples to the [Text Indexes page](https://docs-mongodbcom-staging.corp.mongodb.com/docs/docsworker-xlarge/DOCSP-17260/core/link-text-indexes/)
* Renamed the Text Indexes page to something like "[Perform|Configure] Text Search", to help avoid confusion with the _other_ [Text Indexes](https://docs.mongodb.com/manual/core/index-text/) page.
* Phrase these section titles more like tasks, so it's clearer what users can do. For example, make this first section "Search for an Exact Phrase"